PyAlbany/src/CMakeLists.txt

```diff
@@ -137,6 +137,12 @@ FetchContent_Declare(
 
 FetchContent_MakeAvailable(pybind11)
 
+IF(ALBANY_PYTHON_TESTS)
+  MESSAGE("-- ALBANY_PYTHON_TESTS is TRUE.")
```

Comment:
I think for other tests we use enabled or not enabled instead of True or False, e.g.
`-- Python Tests           NOT Enabled`

Reply:
Do you mean that it would be better to have the `MESSAGE` to use enabled or not enabled to be consistent with the rest?

Reply:
yes, thanks.